applications/sofa/gui/qt/GuiDataRepository.cpp

```diff
@@ -0,0 +1,38 @@
+/******************************************************************************
+*       SOFA, Simulation Open-Framework Architecture, development version     *
+*                (c) 2006-2017 INRIA, USTL, UJF, CNRS, MGH                    *
+*                                                                             *
+* This program is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU General Public License as published by the Free  *
+* Software Foundation; either version 2 of the License, or (at your option)   *
+* any later version.                                                          *
+*                                                                             *
+* This program is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for    *
+* more details.                                                               *
+*                                                                             *
+* You should have received a copy of the GNU General Public License along     *
+* with this program. If not, see <http://www.gnu.org/licenses/>.              *
+*******************************************************************************
+* Authors: The SOFA Team and external contributors (see Authors.txt)          *
+*                                                                             *
+* Contact information: contact@sofa-framework.org                             *
+******************************************************************************/
+/******************************************************************************
+* Contributors:
+*   - damien.marchal@univ-lille1.fr
+*******************************************************************************/
+
+#include "GuiDataRepository.h"
+
+namespace sofa
+{
+namespace gui
+{
+namespace qt
+{
+    sofa::helper::system::FileRepository GuiDataRepository("RUNSOFA_DATA_PATH", "../share/sofa/runSofa/ressources/");
```

Comment:
Why not using current resources dirs (e.g. share/icons)?
BTW, only one S for **resources** ;-)

Reply:
Good question.

This is to make things modular as I see no reason why the icons specific to a given end user application (here runSofa) should be in the data directory of the base framework. Said differently, If the user don't care about runSofa (or runSofa2) why its data repository should be polluated by these resources ?

Reply:
I agree with the modular idea but SofaGuiQt is not runSofa so I think it would better be `sofa::helper::system::FileRepository runSofaDataRepository("RUNSOFA_DATA_PATH", "../share/sofa/runSofa/ressources/");` created in applications/projects/runSofa or `sofa::helper::system::FileRepository GuiDataRepository("GUI_DATA_PATH", "../share/sofa/gui/ressources/");` in applications/sofa/gui.
What do you think?

Reply:
Plugins examples...are in the example sub-directory of the plugin directory and not in share. 
Icon sets of external UI 'project' (like runSofa2) are in their project directory. 
Actually even before this PR the icons of runSofa were not in share but in the runSofa dir...so please don't change that and let's keep a single rule that is clear to understand: specific per 'project' (or 'plugins') files have to stay in their project (plugin) directory.

Reply:
Yep, that's exactly what I was saying in my second post. So what about my proposal?

Reply:
Sorry I miss-understood you last comment. 

I see no problem in making things more consistent by renaming things as long as the files stays separated from the sofa/share.
Can you do that..caus .I'm focused on something else.